### PR TITLE
Add news app to project

### PR DESCRIPTION
**Pull request by raf-lim**
Add app to project. Add temporary welcome url and view.

### CODE REVIEW

File: backend/apps/news/tests.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# from django.test import TestCase
```
Comment (Indoxer) [inline backend/apps/news/tests.py:1]: You can change tests file to tests folder to save consistence with other apps and avoid deleting this file from main branch later.

**Review by Indoxer — COMMENTED** [review]
LGTM

File: backend/apps/news/tests.py (hunk 1 above)
Comment (raf-lim) [reply backend/apps/news/tests.py]: Done.

**Review by Indoxer — APPROVED** [review]
LGTM

**Pull request merged**